### PR TITLE
Script to remove expired quarantined prefixes

### PR DESCRIPTION
**Pull request by MikaKeljaNexer**

### CODE REVIEW

**Comment by MikaKeljaNexer** [comment]
First PR

File: utilities/remove_expired.py [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    Prefixes within prefixes are recursively deleted only if the
+    type is Assigned.
+"""
+
+import ConfigParser
+import os
+import datetime
+# use local pynipap, useful if we are developing
+import sys
+sys.path.append('../pynipap')
```
Comment (garberg) [inline utilities/remove_expired.py:21]: This can be removed

File: utilities/remove_expired.py [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+        }
+    }
+
+    try:
+        res = Prefix.search(query)
+    
+    except socket.error:
+        print >> sys.stderr, "Connection refused, please check hostname and port"
+        sys.exit(1)
+    except xmlrpclib.ProtocolError:
```
Comment (garberg) [inline utilities/remove_expired.py:49]: Hm, won't an authentication error raise some NipapAuthError?

File: utilities/remove_expired.py (hunk 1 above)
Comment (MikaKeljaNexer) [reply utilities/remove_expired.py]: Removed

File: utilities/remove_expired.py (hunk 2 above)
Comment (MikaKeljaNexer) [reply utilities/remove_expired.py]: Yes, I changed to NipapAutError and added also a general exception.

**Comment by MikaKeljaNexer** [comment]
Updated after review comments and also made the script to Python3

**Comment by garberg** [comment]
LGTM, thanks for contributing!

**Pull request merged**